spinalcordtoolbox/deepseg/inference.py

```diff
@@ -115,7 +115,8 @@ def segment_non_ivadomed(path_model, model_type, input_filenames, threshold):
         fnames_out, targets = inference(path_img=fname_in, tmpdir=tmp_create(basename="sct_deepseg"), predictor=net)
         for fname_out, target in zip(fnames_out, targets):
             # TODO: Use API binarization function when output filetype is sct.image.Image
-            run_proc(["sct_maths", "-i", fname_out, "-bin", str(threshold), "-o", fname_out])
+            if threshold is not None:
+                run_proc(["sct_maths", "-i", fname_out, "-bin", str(threshold), "-o", fname_out])
```

Comment:
Great! The rootlets segmentation now works fine (i.e., the prediction is multi-class)! 🎉